web-ng/src/app/shared/models/project.model.ts

```diff
@@ -27,7 +27,7 @@ export class Project {
     readonly layers: Map<string, Layer>
   ) {}
 
-  getLayerForm(layerId: string, formId: string): Form {
+  getForm(layerId: string, formId: string): Form {
```

Comment:
Can we use composition here (`getLayer(layerId)!.getForm(formId)!` instead of combining the two behaviors into one?